tests/typing/pass/ae/types/types_2.ae

```diff
@@ -0,0 +1,3 @@
+
+type 'a list
+type ('a, 'b) map
```

Comment:
This one is probably due to how abstract type declarations are handled in [src/standard/statement.ml](https://github.com/Gbury/dolmen/blob/master/src/standard/statement.ml#L409).
Instead of initializing ```list``` as a parameterized type with a list of type variables as parameters, it is initialized as a function type with the type parameters as types for the functions' arguments.
I think the definition of abstract types should look something like this:
```
type abstract = { id: Id.t; ty: term; vars: term list; loc: location; }
```
instead of: 
```
type abstract = { id: Id.t; ty: term; loc: location; }
```

Reply:
There should be no need to change the type of abstract type definition. If there's any problem it's more likely to be that the wrong abstract type declarations are generated by the parser.

Reply:
I'll take a look

Reply:
I just pushed to your branch a fix for that.

Reply:
Great, Thanks!